docs/man-openmpi/man7/Open-MPI.7.rst

```diff
@@ -0,0 +1,64 @@
+.. _open-mpi:
+.. _mpi:
+
+Open-MPI
```

Comment:
Nit: Open MPI, not Open-MPI.

We try very hard to have consistent branding ("Open MPI", not "OpenMPI", not "Open-MPI", ... etc.).